include/proxy/v4/proxy.ixx

```diff
@@ -0,0 +1,60 @@
+module;
+
+#include <proxy/v4/proxy.h>
+
+export module proxy.v4;
+
+export namespace pro::inline v4 {
+
+using v4::constraint_level;
```

Comment:
Nit: `pro::` should be fine. If we upgrade to `v5` in the future, this code can stay unchanged